src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf8Utility.Transcoding.cs

```diff
@@ -1134,7 +1134,7 @@ public static OperationStatus TranscodeToUtf8(char* pInputBuffer, int inputLengt
                     }
                     else
                     {
-                        pOutputBuffer[0] = (byte)(thisDWord >> 24); // extract [ AA 00 ## ## ]
+                        pOutputBuffer[0] = (byte)(thisDWord >> 16); // extract [ 00 AA ## ## ]
```

Comment:
Thanks for correcting the comment as well! :)

Reply:
:-)